src/common.cpp

```diff
@@ -2866,8 +2866,11 @@ __declspec(dllexport) void *new_dll_graphics_driver(void *game_object)
 	// game-specific initialization
 	if(!ff8)
 		ret = ff7_load_driver(game_object);
-	else
+	else {
+		VOBJ(game_obj, game_object, game_object);
+		VRASS(game_object, current_gfx_driver, 1); // Force software rendering game hacks
```

Comment:
This bit is set here as well https://github.com/julianxhokaxhiu/FFNx/blob/master/misc/FF8.reg#L24-L40, which is suggested to run before using FFNx. If you change the settings on the FF8Config GUI for example the value is going to be overridden though. For Steam, this is always enforced as we hard code the registry value via the dotemu APIs.

Reply:
I will check that thanks. For me the gfx_driver id cannot be enforced by registry in ff8, FFNx forces to 2 anyway, because that's the easiest way to make the game run the custom driver function here

Reply:
The reason I force 1 after is that I found several places in ff8 where current_gfx_driver is testes with the hardcoded value of 1

Reply:
I checked and ""Graphics"=dword:00100021" does not affect the current gfx driver value at all.

And I confirm that FFNx sets this value to "2" itself here: https://github.com/julianxhokaxhiu/FFNx/blob/master/src/common.cpp#L3471

Reply:
Ah ok maybe I remember wrong, so let's change the line over that function to 1 for FF8 at this point if it brings better results visually. Thanks for going this deep, appreciated